README.md

```diff
@@ -193,6 +193,26 @@ class Employee
 end
 ```
 
+Sometimes, system constraints prevent using relation-based scoping. When this happens, you can scope slugs using multiple fields, addressing needs like database structure or performance issues.
+
+Here's a quick setup for multi-field scoping:
+
+```ruby
+class Employee
+  include Mongoid::Document
+  include Mongoid::Slug
+
+  field :name
+  field :company_id
+  field :department_id
+
+  # Scope slug uniqueness by a combination of company and department
+  slug :name, scope: %i[company_id department_id]
+end
+```
+
+Note: this approach creates multiple indexes, differing from single-field scoping, and impacting database performance and storage.
```

Comment:
Hmmm... shouldn't this still create only one _compound_ index? For example:

```
slug :name, scope: %i[company_id department_id]

# Should create
index company_id: 1, department_id: 1, _slugs: 1

# NOT
index company_id: 1, _slugs: 1
index department_id: 1, _slugs: 1
```

Reply:
Sorry for the late response. You're right, it creates only a compound index in this case. I've [fixed it here](https://github.com/mongoid/mongoid-slug/pull/274/commits/601318844bdaf81212754d4212868fb28aca8208) and forgot to remove the note.